package.json

```diff
@@ -9,9 +9,13 @@
     "@emotion/core": "^10.0.28",
     "@emotion/styled": "^10.0.27",
     "emotion-theming": "^10.0.27",
+    "eslint": "^6.8.0",
```

Comment:
Might be better to move these to devDeps - but we can do so in a future PR

Reply:
Yeah true say. Although `gatsby-plugin-eslint` is used at build time it may be necessary in CI?

Reply:
thanks for the review btw :)